src/modules/keyboardmanager/KeyboardManagerEditorLibrary/EditKeyboardWindow.cpp

```diff
@@ -353,6 +351,10 @@ void createEditKeyboardWindow(HINSTANCE hInst, KeyboardManagerState& keyboardMan
 
     // Cannot be done in WM_DESTROY because that causes crashes due to fatal app exit
     xamlBridge.ClearXamlIslands();
+
+    // Calling ClearXamlIslands() outside of the message loop is not enough to prevent
+    // Microsoft.UI.XAML.dll from crashing during deinitialization, see https://github.com/microsoft/PowerToys/issues/10906
+    TerminateProcess(GetCurrentProcess(), 0);
```

Comment:
`KeyboardManagerConstants::EditorWindowEventName` will not be reset which means that remapping will stay suspended.

Reply:
This event is reset when the `event_locker`'s destructor is called in our case it will not be called

Reply:
As a solution, we can move `createEditKeyboardWindow` implementation into a separate function

Reply:
Or if it's simpler just invoke the events explicitly.

Reply:
To reset it explicitly we need to introduce a new method in `event_locker`. I would prefer not to do it as it would not be useful anywhere else.

Reply:
OK